VerticalMetrics/README.md

```diff
@@ -28,25 +28,39 @@ This rule can be voided if a font is being upgraded and previously had inconsist
 
 *For brevity, we'll refer to the 3 sets of metrics as Typo, Hhea, Win.* 
 
-### 4. Win Ascent and Win Descent values must be the same as the family's tallest/deepest yMin and yMax bbox values
+### 4. Win Ascent and Win Descent values must be the same as the family's tallest/deepest yMin and yMax bbox values (rounded up)
```

Comment:
What does rounded up mean in this case?

Reply:
if typoAscent = 1116 > it has to be 1120
if typoDescent = -1116 > it has to be -1120

I know it can be confusing, i didn't find a better simple expression.

Reply:
wait no I am talking about WIN values, and there are both positive values, so round up still make sense.
if winAscent = 1116 > it has to be 1120
if winDescent = 254 > it has to be 260

Reply:
What's the benefit of doing this?

Reply:
To avoid bad scaling at small sizes, with point being out of win-metrics boundaries.

Reply:
Sorry if I'm being dense here but I don't know what this means. Do you have an example?

Reply:
If winAscent=y-max, there is a chance that top of glyphs is still cropped in browsers at small sizes. This was recommended by hinting specialists at my previous job, and it is also appearing in the glyphs' tuto (but with no explanations). I am trying to replicate and I can't. My guess is that it is a conservative/oldschool QA behaviour for rendering on low definition screens. 

I don't mind dropping this recommendation.

Reply:
In general when I hint diacritics, I tend to lock the bottom of the glyph to the whole pixel and hint 'upwards' (thus ensuring separation between diacritic and base glyph). This will tend to extend the diacritic upwards and can cause it to 'round' up to the whole pixel for greater clarity / contrast.

My suspicion is that this recommendation is intended to help account for that and avoid possible clipping should the hinted glyph extend past the winAscent value at small sizes on low resolution screens.

Now, given that pretty much none of the fonts on GF are hand-hinted, maybe it doesn't matter (I have no idea how ttfautohint does things), but it might not be a bad idea to round up.

Reply:
Thanks! I now understand the issue.

I've never encountered this myself but it could've been an issue in the GDI days but again I'm only speculating here.

Reply:
Yeah, I haven’t encountered it myself either :). Hinting was much more heavy handed in the old days, so I can see how it might have had a more noticeable impact.